README.md

```diff
@@ -298,17 +299,21 @@ Parameters affect the de-identification methods specified in the type rules and
 You can specify dateShift as a de-identification method in the configuration file. With this method, the input date/dateTime/instant value will be shifted within a 100-day differential. The following algorithm is used to shift the target dates:
 
 ### Input
-* A date/dateTime/instant value (required)
-* FHIR resource id (optional). If not specified, an empty string will be used.
-* Date shift key (optional). If not specified, a randomly generated string will be used.
+- [Required] A date/dateTime/instant value
+- [Optional] _dateShiftKey_. If not specified, a randomly generated string will be used as default key.
+- [Optional] _dateShiftScope_. If not specified, _resource_ will be set as default scope.
 
 ### Output
 * A shifted date/datetime/instant value
 
 ### Steps
-1. Create a string by combining FHIR resource id and date shift key.
-2. Feed the above string to hash function to get an integer between [-50, 50]. 
-3. Use the above integer as the offset to shift the input date/dateTime/instant value
+1. Get _dateShiftKeyPrefix_ according to _dateShiftScope_.
+- For scope _resource_, _dateShiftKeyPrefix_ refers to the resource ID.
+- For scope _file_, _dateShiftKeyPrefix_ refers to the file name.
+- For scope _folder_, _dateShiftKeyPrefix_ refers to the input folder name.

```

Comment:
Do we use folder name or <folder name + relative path>? What happens if there are two subfolders with the same name and the user is using -r flag?

Reply:
We use <folder name> here, which is the root input folder provided by user. If the user is using -r flag, files will be processed recursively, and all dates will be shifted with same amount, no matter which subfolder they belongs to. This scope provides users a way to shift all dates in the input data set.